draft-ietf-quic-recovery.md

```diff
@@ -289,8 +289,15 @@ be subtracted from the RTT as long as the result is larger than the min_rtt.
 If the result is smaller than the min_rtt, the RTT should be used, but the
 ack delay field should be ignored.
 
-Like TCP, QUIC calculates both smoothed RTT and RTT variance similar to those
-specified in {{?RFC6298}}.
+A sender calculates both smoothed RTT and RTT variance similar to those
+specified in {{?RFC6298}}, see {{on-ack-received}}.
+
+A sender might take multiple samples per RTT because ACK frames that are
```

Comment:
Suggestion: "A sender will take multiple samples per RTT when ACK frames are received within an RTT that acknowledge larger packet numbers than before("

Reply:
A sender will take multiple samples per RTT.  RTT samples are taken when the largest acknowledged packet number is increased by an acknowledgment (see).